### PR TITLE
updates build.gradle to new CoFH

### PR DESCRIPTION
**Pull request by way2muchnoise**
also makes it more conform

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -17,6 +17,7 @@
 *.xml
 .cache
 lib/
```
Comment (jakimfett) [inline .gitignore]: If everything is converting to libs shouldn't this line get removed?

File: build.gradle [hunk 2]
```diff
@@ -153,8 +153,8 @@ task getLibraries {
 		}
 	}
 
-        def cofhSource = 'http://minecraft.curseforge.com/mc-mods/220333-cofhlib/files/2225609/download'
-        def cofhDest = new File('lib', 'CoFHLib-1.7.10-1.0.0B9-68-dev.jar')
+    def cofhSource = 'http://minecraft.curseforge.com/mc-mods/220333-cofhlib/files/2228470/download'
+    def cofhDest = new File('libs', 'CoFHLib-[1.7.10]1.0.0B9-118-dev.jar')
```
Comment (jakimfett) [inline build.gradle]: Blah. Do we really need the parenthesis in the filename? Really?

Comment (way2muchnoise) [inline build.gradle]: It's just what it is in the v6 branch want me to remove it?

Comment (jakimfett) [inline build.gradle]: Yes please, I'd already removed them for the buildfile earlier.

**Pull request merged**